vendor/github.com/OpenBazaar/go-ethwallet/wallet/wallet.go

```diff
@@ -563,7 +563,10 @@ func (wallet *EthereumWallet) ChainTip() (uint32, chainhash.Hash) {
 	if err != nil {
 		return 0, *h
 	}
-	h, _ = chainhash.NewHashFromStr(hash.Hex()[2:])
+	h, err = util.CreateChainHash(hash.Hex())
+	if err != nil {
+		log.Error(err)
+	}
 	return num, *h
```

Comment:
It would seem that the empty string chainhash is returned to handle this return properly. I think it would be better for the ChainTip method to handle creation of the empty string chainhash instead of forcing that behavior on all of the methods which call CreateChainHash. Further, this goes against go convention where a non-nil error being returned is also providing a valid response as well. Typically if a non-nil error comes back, we want to ignore the result. It should be the same here, IMO.